lib/danger/ci_source/jenkins.rb

```diff
@@ -5,24 +5,27 @@ module Danger
   # https://jenkins-ci.org
 
   # ### CI Setup
-  # Ah Jenkins, so many memories. So, if you're using Jenkins, you're hosting your own environment. You
+  # Ah Jenkins, so many memories. So, if you're using Jenkins, you're hosting your own environment.
   #
   # #### GitHub
   # You will want to be using the [GitHub pull request builder plugin](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+request+builder+plugin)
   # in order to ensure that you have the build environment set up for PR integration.
   #
   # With that set up, you can edit your job to add `bundle exec danger` at the build action.
   #
-  # ### GitLab
-  #
-  # You will want to be using the [GitLabe Plugin](https://github.com/jenkinsci/gitlab-plugin)
-  # in order to ensure that you have the build environment set up for PR integration.
+  # #### GitLab
+  # You will want to be using the [GitLab Plugin](https://github.com/jenkinsci/gitlab-plugin)
+  # in order to ensure that you have the build environment set up for MR integration.
   #
   # With that set up, you can edit your job to add `bundle exec danger` at the build action.
   #
   # ### Token Setup
   #
-  # As you own the machine, it's up to you to add the enviroment variable for the `DANGER_GITHUB_API_TOKEN`.
+  # #### GitHub
```

Comment:
This may not be what you meant, but I figured this was better than going to the 5th header level, I can change it if you disagree.